tests/defences/trainer/test_adversarial_trainer_madry_pgd.py

```diff
@@ -0,0 +1,104 @@
+# MIT License
```

Comment:
Please remove this file or move the entire test to pytest in a separate PR..

Reply:
moving to separate PR